File: 151003/Patiyuk/TaskREST/src/main/java/by/bsuir/taskrest/entity/Creator.java
package by.bsuir.taskrest.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Creator implements BaseEntity<Long> {
    private Long id;
    private String login;
    private String password;
    private String firstName;
    private String lastName;
}

File: 151003/Alekseeva/TaskREST/TaskREST/Models/Creator.cs
using System.ComponentModel.DataAnnotations;

namespace TaskREST.Models;

public class Creator
{
    public long Id { get; set; }

    [MaxLength(64)] public string Login { get; set; } = string.Empty;

    [MaxLength(128)] public string Password { get; set; } = string.Empty;

    [MaxLength(64)] public string FirstName { get; set; } = string.Empty;

    [MaxLength(64)] public string LastName { get; set; } = string.Empty;
}